benchmark/benchmark_tridiagonal_solver.cpp

```diff
@@ -0,0 +1,108 @@
+#include "mkl_lapacke.h"
+
+#include <tuvx/linear_algebra/linear_algebra.hpp>
+
+#include <benchmark/benchmark.h>
+
+#include <mkl_lapack.h>
```

Comment:
we probably do not need this header as it is for Fortran.

Reply:
@sjsprecious I fixed this in my latest commit.